flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -981,16 +998,25 @@ public boolean isBounded() {
                 case "NewSource":
                     checkArgument(
                             !failingSource, "Values Source doesn't support as failing new source.");
+                    final FinishingLogic finishingLogic =
+                            finite ? FinishingLogic.FINITE : FinishingLogic.INFINITE;
+                    final Boundedness boundedness =
```

Comment:
do we have a check somewhere that infinite + bounded makes no sense?

Reply:
added